### PR TITLE
add vuepress plausible plugin

### PR DESCRIPTION
**Pull request by cmunns**

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
- No markdown files were changed, so no page checks were run!

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -25,8 +25,8 @@ module.exports = {
       md.use(require('markdown-it-footnote'))
       md.use(require('markdown-it-task-lists'))
       md.use(require('markdown-it-deflist')),
-      md.use(require('markdown-it-imsize')),
-      md.use(require('markdown-it-image-lazy-loading'))
+        md.use(require('markdown-it-imsize')),
+        md.use(require('markdown-it-image-lazy-loading'))
```
Comment (drewdelano) [inline docs/.vuepress/config.js:29]: weird whitespace formatting?  maybe this is a tabs and spaces situation?

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,18289 @@
 {
   "name": "nft-school",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (drewdelano) [inline package-lock.json:4]: the lockfile change here feels pretty substantial

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
- No markdown files were changed, so no page checks were run!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
- No markdown files were changed, so no page checks were run!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build failed...

```

> nft-school@2.0.0 docs:build
> NODE_OPTIONS=--openssl-legacy-provider vuepress build docs

node: --openssl-legacy-provider is not allowed in NODE_OPTIONS
```
- No markdown files were changed, so no page checks were run!

**Review by yusefnapora — CHANGES_REQUESTED** [review]
Hmm, on second thought... the "filecorgi" CI script is still apparently running on node 16, which chokes with the `--openssl-legacy-provider` flag.

So maybe the thing to do here is try reverting back to node 16 and lockfile v1, so we have a consistent env everywhere. I'll try that and see if that sorts things out.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
- No markdown files were changed, so no page checks were run!

**Comment by yusefnapora** [comment]
lol, okay. Fleek is dead set on using node 18, apparently, so I brought back the fixes and just made the CI script also use node 18. 🤞

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
- No markdown files were changed, so no page checks were run!

**Review by yusefnapora — APPROVED** [review]
Okay, everything seems to be building again :) @cmunns feel free to merge if you're alright with the changes I made.

**Pull request merged**